openfpga_flow/regression_test_scripts/basic_reg_test.sh

```diff
@@ -9,182 +9,185 @@ PYTHON_EXEC=python3.8
+run-task basic_tests/k4_series/k4n4_frac_mult $@
 echo -e "Testing K4N5 with pattern based local routing";
-run-task basic_tests/k4_series/k4n5_pattern_local_routing --debug --show_thread_logs
+run-task basic_tests/k4_series/k4n5_pattern_local_routing $@
 
 echo -e "Testing different tile organizations";
 echo -e "Testing tiles with pins only on top and left sides";
-run-task basic_tests/tile_organization/top_left_custom_pins --debug --show_thread_logs
+run-task basic_tests/tile_organization/top_left_custom_pins $@
 echo -e "Testing tiles with pins only on top and right sides";
-run-task basic_tests/tile_organization/top_right_custom_pins --debug --show_thread_logs
+run-task basic_tests/tile_organization/top_right_custom_pins $@
 echo -e "Testing tiles with pins only on bottom and right sides";
-run-task basic_tests/tile_organization/bottom_right_custom_pins --debug --show_thread_logs
+run-task basic_tests/tile_organization/bottom_right_custom_pins $@
 echo -e "Testing tiles with I/O in center grid";
-run-task basic_tests/tile_organization/tileable_io --debug --show_thread_logs
+run-task basic_tests/tile_organization/tileable_io $@
 
 echo -e "Testing global port definition from tiles";
-run-task basic_tests/global_tile_ports/global_tile_clock --debug --show_thread_logs
-run-task basic_tests/global_tile_ports/global_tile_reset --debug --show_thread_logs
-run-task basic_tests/global_tile_ports/global_tile_4clock --debug --show_thread_logs
-run-task basic_tests/global_tile_ports/global_tile_4clock_pin --debug --show_thread_logs
+run-task basic_tests/global_tile_ports/global_tile_clock $@
+run-task basic_tests/global_tile_ports/global_tile_reset $@
+run-task basic_tests/global_tile_ports/global_tile_4clock $@
+run-task basic_tests/global_tile_ports/global_tile_4clock_pin $@
 
 echo -e "Testing configuration chain of a K4N4 FPGA using .blif generated by yosys+verific";
-run-task basic_tests/verific_test --debug --show_thread_logs
+run-task basic_tests/verific_test $@
 
 echo -e "Testing explicit multi verilog files";
-run-task basic_tests/explicit_multi_verilog_files --debug --show_thread_logs
-
-echo -e "Testing output files without time stamp";
-run-task basic_tests/no_time_stamp --debug --show_thread_logs
-# Run git-diff to ensure no changes on the golden netlists
-if git diff --name-status -- ':openfpga_flow/tasks/basic_tests/no_time_stamp/golden_outputs_no_time_stamp/**'; then
-  echo -e "Golden netlist remain unchanged"
-else
-  echo -e "Detect changes in golden scripts"; exit 1;
-fi
+run-task basic_tests/explicit_multi_verilog_files $@
 
 echo -e "Test the remove of runtime directories"
-run-task basic_tests/explicit_multi_verilog_files --debug --show_thread_logs --remove_run_dir all
-
-# Repgression test to test multi-user enviroment
-cp -r */*/basic_tests/full_testbench/configuration_chain /tmp/
-cd /tmp/ && run-task configuration_chain --debug --show_thread_logs
+clear-task-run basic_tests/explicit_multi_verilog_files $@
```

Comment:
@ganeshgore This is where the ``clear-task-run`` is used. It is to avoid any duplicated options, i.e., ``--remove_run_dir all``